--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -936,6 +936,19 @@ enum flow_offload_type {
 	FLOW_OFFLOAD_DEL,
 };
 
+/* QCA RFS Support - Start */
+#ifdef CONFIG_RFS_ACCEL
+typedef int (*set_rfs_filter_callback_t)(struct net_device *dev,
+					 __be32 src,
+					 __be32 dst,
+					 __be16 sport,
+					 __be16 dport,
+					 u8 proto,
+					 u16 rxq_index,
+					 u32 action);
+#endif
+/* QCA RFS Support - End */
+
 /*
  * This structure defines the management hooks for network devices.
  * The following hooks can be defined; unless noted otherwise, they are
